tests/system/action/poll/test_anonymize.py

```diff
@@ -45,3 +47,109 @@ def test_anonymize(self) -> None:
         assert user.get("vote_$1_ids") == []
         assert user.get("vote_delegated_vote_$_ids") == ["1"]
         assert user.get("vote_delegated_vote_$1_ids") == []
+
+
+class AnonymizePoll(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.create_model(
+            "assignment/1",
+            {
+                "title": "test_assignment_QLydMOqkyOHG68yZFJxl",
+                "open_posts": 1,
+                "candidate_ids": [1],
+            },
+        )
+        self.create_model(
+            "poll/1",
+            {
+                "content_object_id": "assignment/1",
+                "title": "test_title_3LbUCNirKirpJhRHRxzW",
+                "pollmethod": "YNA",
+                "type": Poll.TYPE_NAMED,
+                "state": Poll.STATE_FINISHED,
+                "option_ids": [11],
+                "meeting_id": 113,
+                "voted_ids": [1, 2],
+                "votesvalid": "2.000000",
+                "votesinvalid": "0.000000",
+                "votescast": "2.000000",
+            },
+        )
+        self.create_model(
+            "option/11",
+            {
+                "meeting_id": 113,
+                "poll_id": 1,
+                "yes": "1.000000",
+                "no": "1.000000",
+                "abstain": "0.000000",
+                "vote_ids": [1, 2],
+            },
+        )
+        self.create_model("meeting/113", {"name": "my meeting"})
+
+        self.create_model(
```

Comment:
vote models also have the `delegate_user_id` field. this needs to be cleared.

Reply:
Do you mean the `delegated_user_id`? It is cleared.
Update the anonymize tests. I use one class and add some assert for the delegated_user_id.